add-on/panels/css/panel.css

```diff
@@ -775,3 +784,37 @@ img.fte-hello-web-share {
   background-color: #5cccee;
   color: #fff;
 }
+
+.share-panel-container > .share-panel-overlay {
+  background-color: rgba(0, 0, 0, .3);
+  bottom: 0;
+  display: none;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.share-panel-container > .room-invitation-overlay {
```

Comment:
Testing it out, I'm not seeing the css being applied in the panel. I think it needs moving from conversation.css to probably a new css file that's shared just for desktop.